vagrant/provision-master.sh

```diff
@@ -5,6 +5,11 @@ source $(dirname $0)/provision-config.sh
 
 OPENSHIFT_SDN=$4
 
+NETWORK_CONF_PATH=/etc/sysconfig/network-scripts/
+sed -i '/NM_CONTROLLED=no/d' ${NETWORK_CONF_PATH}ifcfg-eth1
```

Comment:
Maybe better to comment out instead of delete?

And should probably start with ^